frontend/shared/src/components/forms/inputs/TextInput.sc.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  NumberInputProps,
+  TextAreaProps,
+  TextInput,
+  TextInputProps,
+} from 'hds-react';
+import styled from 'styled-components';
+
+export type $TextInputProps = {
+  $type: 'text' | 'decimal' | 'number' | 'textArea';
+};
+
+export const $TextInput = styled(TextInput)<
+  $TextInputProps & (NumberInputProps | TextInputProps | TextAreaProps)
+>`
+  ${(props) =>
+    !props.errorText ? `margin-bottom: ${props.theme.spacing.m};` : ''}
+  ${(props) =>
+    props.$type === 'textArea'
+      ? `&[style] {
+          height: 208px !important;
```

Comment:
Why don't you leave the height flexible to the parent? Wouldn't `min-height` do without `!important`?

Reply:
This is old moved code but I tried to removed it and didnt have effect on text area component